rutil/Data.hxx

```diff
@@ -1126,6 +1148,35 @@ bool operator<(const Data& lhs, const Data& rhs);
 bool operator<(const Data& lhs, const char* rhs);
 bool operator<(const char* lhs, const Data& rhs);
```

Comment:
It will be nice to add these kinds of operators for new extensions.
It might help in case resip::Data is an item of a container.

Reply:
I've added some other operators, but have not added the  < and > yet, usually, I do not use them :).